packages/wp-story-editor/src/api/pageTemplate.js

```diff
@@ -33,8 +37,12 @@ export function getCustomPageTemplates(config, page = 1) {
   return apiFetch({ path }).then(({ headers, body }) => {
     const totalPages = parseInt(headers['X-WP-TotalPages']);
     const templates = body.map((template) => {
-      return { ...template['story_data'], templateId: template.id };
+      const storyData = template['story_data'];
+      storyData.elements = storyData.elements.map(snakeToCamelCaseObjectKeys);
```

Comment:
Is there a need to specifically camelcase everything inside story_data? 🤔

Reply:
On the latest `main` branch, I am still getting snake case as a response here.

Some more details would be very helpful.

![image](https://user-images.githubusercontent.com/6297436/142394647-64b76bdd-9bc2-44ff-8e10-d553efac83a9.png)

Reply:
The response of a newly saved custom template ☝️

Reply:
But surely not with this branch, no?

Since you've added camelCase to `getResourceFromAttachment`, all resources and their image sizes should be camelCased already.

Reply:
Oh, but not for existing templates.

Let me leave a comment on the PR itself.

Reply:
> But surely not with this branch, no?

I see, yes, happening on this branch for new templates because of adding `getResourceFromAttachment` to media. 

> Oh, but not for existing templates.

Correct not for existing templates.